src/avram/object_extensions.cr

```diff
@@ -1,4 +1,16 @@
 class Object
+  def blank?
```

Comment:
was this removed from the standard lib?

Reply:
It never existed in standard lib on Object from what I can tell; only on String. We had to use a shard to get it on all objects. Since the implementation was small, I figured it was easier to just bring it over.